simulation/simple_sensor_simulator/src/simple_sensor_simulator.cpp

```diff
@@ -205,6 +218,7 @@ void ScenarioSimulator::spawnMiscObjectEntity(
   simulation_api_schema::SpawnMiscObjectEntityResponse & res)
 {
   misc_objects_.emplace_back(req.parameters());
+  entity_status_.insert({req.parameters().name(), simulation_api_schema::EntityStatus{}});
```

Comment:
Why do you initialize with initial value?
I think this value should be initialized from value `req`.

Reply:
`entity_status_` is `std::map` for [EntityStatus](https://github.com/tier4/scenario_simulator_v2/blob/11c3ea1998edeb80eae2eb43b9e4cccc7c3a9bb3/simulation/simulation_interface/proto/simulation_api_schema.proto#L12-L19) with entity `name` as `key`. 
At the time of spawning, I add an entity to this map to only indicate that such an object exists, and leave its status empty because it will be updated. 
The values in this map are not used before update entities status. 
But of course I can develop the [EntityStatus](https://github.com/tier4/scenario_simulator_v2/blob/11c3ea1998edeb80eae2eb43b9e4cccc7c3a9bb3/simulation/simulation_interface/proto/simulation_api_schema.proto#L12-L19)  initialization for each spawn operation.
([spawn requests](https://github.com/tier4/scenario_simulator_v2/blob/11c3ea1998edeb80eae2eb43b9e4cccc7c3a9bb3/simulation/simulation_interface/proto/simulation_api_schema.proto#L115-L121) don't have simple fields that can be rewritten to [EntityStatus](https://github.com/tier4/scenario_simulator_v2/blob/11c3ea1998edeb80eae2eb43b9e4cccc7c3a9bb3/simulation/simulation_interface/proto/simulation_api_schema.proto#L12-L19) , but I should be able to handle it)

Reply:
I developed a method `insertEntitySpawnedStatus` that fills and adds a new `entity_status_` during spawning.